File: program/convert-weeks-to-hours/convert_weeks_to_hours.dart
import 'dart:io';
 
void main() {
  print('Enter Number of Weeks : ');
  int num_of_weeks = int.parse(stdin.readLineSync()!);
 
  int total_hrs = num_of_weeks * 168;
 
  print('Total hours : \n$total_hrs');
}

File: program/convert-weeks-to-hours/convert-weeks-to-hours.jl
result = 0
  
# Prompt to enter
total_hrs = 0

println("Enter number of weeks : ")

num_of_weeks = readline()
num_of_weeks = parse(Int64, num_of_weeks) 
  
total_hrs = num_of_weeks * 168

  
println("Total hrs :", total_hrs)
